addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -446,8 +446,9 @@ class Im_LivechatChannelRule(models.Model):
     _description = 'Livechat Channel Rules'
     _order = 'sequence asc'
 
-    regex_url = fields.Char('URL Regex',
-        help="Regular expression specifying the web pages this rule will be applied on.")
+    regex_url = fields.Char("URL Regex", required=True, default="/",
```

Comment:
Note: required flag is only useful in view, if we want to really enforce it we should have constraints.